recipes/r-rcppdate/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set version = '0.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rcppdate
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/RcppDate_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RcppDate/RcppDate_{{ version }}.tar.gz
+  sha256: 117721fc677dfb4209200a7ff894fbbb8ee1b652d01b3878b11c3253733b4a5f
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('RcppDate')"           # [not win]
+    - "\"%R%\" -e \"library('RcppDate')\""  # [win]
+
+about:
+  home: https://github.com/eddelbuettel/rcppdate
+  license: GPL-2
```

```suggestion
  license: GPL-2.0-or-later
```
GPL (>= 2) -> https://spdx.org/licenses/GPL-2.0-or-later.html

---

Done, thanks for the feedback.